quickstarts/mssql-queryplan/config.yml

```diff
@@ -0,0 +1,31 @@
+# Name of the quickstart (required)
+name: nri-mssql-experimental
+title: Microsoft SQL Server with Query Plan logging
+description: |
+  The original SQL Server nri with added Query Plan logging
+
+summary: |
+  Send SQL Server Query Plans to the New Relic Logging API
+
+logo: logo.svg
+# Support level: New Relic | Verified | Community (required)
+level: Community
+
+website: https://cloud.google.com/apigee
+
+keywords:
+  - nrlabs
+  - nrlabs-data
+  - database
+  - infrastructure
+
+# Authors of the quickstart (required)
+authors:
+  - New Relic Labs
+
+documentation:
+  - name: Apigee Distributed Tracing documentation
+    description: |
+      Installation and configuration instructions
+    url: https://github.com/newrelic-experimental/nri-mssql-experimental
```

Comment:
Is there a reference to a docs.newelic.com page you can use here, along with a link to the repository?

Reply:
@jpvajda We haven't been adding our newrelic-experimental projects to doc.newrelic.com as they're unsupported and not "Product"